pyproject.toml

```diff
@@ -13,3 +13,12 @@ asyncio_mode = "auto"
 testpaths = [
   "test",
 ]
+
+[[tool.mypy.overrides]]
+module = [
+    "batchspawner",
+    "jupyterhub.tests.utils",
+    "jupyterhub.utils",
+    "setuptools",
```

Comment:
This allows to run `mypy` also on `setup.py` and `test/`, so it boils down to `mypy .`